test/test_linalg.py

```diff
@@ -1638,7 +1638,12 @@ def run_test_case(input, p):
         a = torch.eye(3, dtype=dtype, device=device)
         a[-1, -1] = 0  # make 'a' singular
         for p in norm_types:
-            run_test_case(a, p)
+            try:
+                run_test_case(a, p)
+            except numpy.linalg.LinAlgError:
```

Comment:
numpy is imported as `np`, so this should be `np.linalg.LinAlgError`

Reply:
ooops!